pkg/virt-operator/util/types.go

```diff
@@ -40,6 +40,7 @@ type Stores struct {
 	InstallStrategyJobCache       cache.Store
 	InfrastructurePodCache        cache.Store
 	PodDisruptionBudgetCache      cache.Store
+	ServiceMonitorCache           cache.Store
```

Comment:
This store needs to be initialized. And you need to check if the `ServiceMonitor` CRD exists. We do something similar for SCCs here: https://github.com/kubevirt/kubevirt/blob/d6832ae4e1802aa8c35d46c27d73059bdb0ea7ad/pkg/virt-operator/application.go#L166